WORKSPACE

```diff
@@ -140,7 +140,7 @@ maybe(
 
 load("@rules_foreign_cc//foreign_cc:repositories.bzl", "rules_foreign_cc_dependencies")
 
-rules_foreign_cc_dependencies(make_version = "4.2")
+rules_foreign_cc_dependencies(make_version = "4.3")
```

Comment:
Please remove this change before merging!

Reply:
Done!